ui/app/components/app-form/project-settings.ts

```diff
@@ -208,10 +221,20 @@ export default class AppFormProjectSettings extends Component<ProjectSettingsArg
     dataSource.setGit(git);
     ref.setDataSource(dataSource);
     ref.setDataSourcePoll(dataSourcePoll);
+
+    if (this.serverHcl && this.project.waypointHcl) {
+      let hclEncoder = new window.TextEncoder();
+      let waypointHcl = hclEncoder.encode(this.project.waypointHcl);
+      // Hardcode hcl for now
+      ref.setWaypointHclFormat(0);
```

Comment:
Seeing `this.project.waypointHclFormat` being set to `0` up above (with the constant) should this be using that var instead of setting the value here?

Reply:
updated it to use a `FORMAT` constant for clarity

Reply:
Nice makes sense.